### PR TITLE
Use defined input information for action

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
https://eaflood.atlassian.net/browse/FCEP-84

### CODE REVIEW

File: test/models/actions-inputs-model.test.js [hunk 1]
```diff
@@ -1,61 +1,79 @@
...
+        unit: 'hectares',
+        description: 'winter crops area'
+      }
+
+    }
+  }
   test('if error provided, should populate errorMessage', () => {
     const error = 'sample error message'
-    const { model } = actionsInputsModel('', {}, error)
+    const { model } = actionsInputsModel('SD74445738', actions.fg6, error)
```
Comment (jaucourt) [inline test/models/actions-inputs-model.test.js]: I'd have been tempted to leave the parcel ref empty, as we're not asserting anything on that, we're focused on the error. The same goes for all the subsequent statements where the parcel ref is unimportant.

File: test/models/actions-inputs-model.test.js [hunk 2]
```diff
@@ -1,61 +1,79 @@
...
-    expect(model.label.text).toBe('Enter the area in hectares')
-    expect(model.hint.text.trim()).toBe(`What is the area you wish to claim on in hectares for ${parcelRef}?`)
+    const { model } = actionsInputsModel(parcelRef, actions.sw6)
+    expect(model.label.text).toBe('Enter the winter crops area in hectares')
+    expect(model.hint.text.trim()).toBe(`What is the winter crops area you wish to claim for in hectares for ${parcelRef}?`)
   })
 
   test('Parcel data is included in the model when a parcel is included', () => {
-    const { parcelModel } = actionsInputsModel('AAA', { id: 'FG1' }, '', allParcels)
+    const parcelModel = actionsInputsModel('SD74445738', actions.fg6, '', allParcels).parcelModel
```
Comment (jaucourt) [inline test/models/actions-inputs-model.test.js]: Following the convention above where we're using object destructuring, you can do:
```
const { parcelModel } = actionsInputsModel('SD74445738', actions.fg6, '', allParcels)
```

**Review by jaucourt — CHANGES_REQUESTED** [review]
Couple of tiny things, one of which is a matter of opinion so disregard it if you choose. Nice work :)

File: test/models/actions-inputs-model.test.js (hunk 2 above)
Comment (neilbmclaughlin) [reply test/models/actions-inputs-model.test.js]: Good spot.

File: test/models/actions-inputs-model.test.js (hunk 1 above)
Comment (neilbmclaughlin) [reply test/models/actions-inputs-model.test.js]: Good spot. Removed.

**Review by jaucourt — APPROVED** [review]

**Pull request merged**